srcpkgs/supersonic-desktop/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'supersonic-desktop'
+pkgname=supersonic-desktop
+version=0.8.2
+revision=1
+build_style=go
+go_import_path="github.com/dweymouth/supersonic"
+hostmakedepends="pkg-config gcc"
+makedepends="libglvnd-devel mpv-devel libX11-devel libappindicator-devel libsecret-devel libXxf86vm-devel" 
+depends="libsecret libappindicator"
+short_desc="Full-featured, mpv-based desktop client written in go for Subsonic and Jellyfin servers"
+maintainer="Michał Szmidt <michal.sz@disroot.org>"
+license="GPL-3.0"
+homepage="https://github.com/dweymouth/supersonic"
+distfiles="https://github.com/dweymouth/supersonic/archive/refs/tags/v${version}.tar.gz"
+checksum=55ce49566d3350fe789934310509763dc2646ea09cc5c8d364daca49d692b185
+
+post_build() {
+	# go install command has no possibility to change output file name
+	# https://github.com/golang/go/issues/47176
+	# There is confilicting supersonic package on GitHub, see repology
+	mv ${GOPATH}/bin/supersonic ${GOPATH}/bin/supersonic-desktop
+}
+
+post_install() {
+	mv res/appicon.png res/supersonic-desktop.png
```

Comment:
use the vinstall rename argument (see manual.md)

Reply:
Done

`vinstall res/appicon.png 644 usr/share/pixmaps supersonic-desktop.png`